department-of-veterans-affairs/caseflow
# Paula/caseflow 1126 judge tabs

Resolves [# 10044](https://github.com/department-of-veterans-affairs/caseflow/issues/10044)
Jira ticket [Caseflow-1126](https://vajira.max.gov/browse/CASEFLOW-1126)

### Description
Judges' individual queues only show a single table that is the equivalent of attorney and generic queues' "assigned" tabs. Judges can navigate to their "assign cases" view using the dropdown menu in the top right of their queue which provides a near-equivalent of the "on hold" tab. However, since judges now have the ability to assign admin actions directly to the VLJ support staff there are some on hold tasks that are not represented in the "assign cases" view.

This ticket exists to add tabs for "on hold" and "completed" tasks (in addition to the existing "assigned" table view they see now). Also confirmed that acting Judges that currently see the attorney queue tabs should continue to see the attorney queue tabs since the majority of their work is still attorney-centered activities.

### Acceptance Criteria
- [x] Judges should now see "On Hold" and "Completed" tabs when signing into queue. Of Note: Attorney version of the queue tabs displays a "Veteran Documents" link that the Judges do not have; Judge version of the queue tabs displays a "Document ID" column that the Attorney version does not have.
- [x] Acting judges should still see the same Attorney-version of the queue tabs.

### Testing Plan
1. Login to queue as a judge (BVAAABSHIRE)
2. Notice that there are now 3 queue tabs under "Your cases":  Assigned (which they already had), On Hold, and Completed.
3. Notice that there is no column for "Veteran Documents" and that there is a column for "Document ID"
3. Login to queue as an attorney (BVASCASPER1)
4. Notice that there are also 3 tabs. No code was changed here.  Just observe that there is no "Document ID" column but there is a "Veterans Documents" column for comparing with an acting VLJ.
5. Login to queue as an acting VLJ (BVAACTING)
6. Notice that there are also 3 tabs.  Notice that the acting VLJ's columns are the same as those for an attorney.

### User Facing Changes
#### Judge Screenshots
**BEFORE**
<img width="590" alt="Judge Your Cases View - no tabs" src="https://user-images.githubusercontent.com/23639439/114941642-691be300-9e11-11eb-9525-fa10bc2167ba.png">

**AFTER**
_Assigned Tab_
<img width="590" alt="Jude Cases Assigned tab" src="https://user-images.githubusercontent.com/23639439/114941713-7f29a380-9e11-11eb-99a4-269db084c8bd.jpg">

_On Hold tab_
<img width="590" alt="Judge Cases On Hold tab" src="https://user-images.githubusercontent.com/23639439/114941724-8486ee00-9e11-11eb-9f34-ad1e1e7890e3.jpg">

_Completed tab_
<img width="590" alt="Judge Cases Completed tab" src="https://user-images.githubusercontent.com/23639439/114941753-8badfc00-9e11-11eb-8340-eca18f45bb8e.jpg">

####Acting VLJ Screenshot (no changes, just for reference)
<img width="590" alt="Acting Judge tabs" src="https://user-images.githubusercontent.com/23639439/114942374-71285280-9e12-11eb-8b2f-31ed6e6a5cf4.jpg">

####Attorney Screenshot (no changes, just for reference)
<img width ="590" alt="Attorney tabs" src="https://user-images.githubusercontent.com/23639439/114942507-a16ff100-9e12-11eb-8bf0-b14b5600158e.jpg">





### Code Documentation Updates
